ws/ShardManager.ts

```diff
@@ -0,0 +1,35 @@
+import DiscordWebsocket, { ConnectionProperties } from './DiscordWebsocket';
+
+interface BotGateway {
+  session_start_limit: {
+    max_concurrency: number;
+    remaining: number;
+    reset_after: number;
+    total: number;
+  };
+  shards: number;
+  url: string;
+}
+
+export default class ShardManager {
+  private shards: Map<number, DiscordWebsocket> = new Map();
```

Comment:
No private
```suggestion
  shards: Map<number, DiscordWebsocket> = new Map();
```

Reply:
I'm still fairly new to how a shard manager works. Where else (outside of the class) could that property be used?

Reply:
Being used outside of the class is the reason why it shouldn't be private

Reply:
I was just curious if it was going to be used outside of the class.

Reply:
There are reasons why people want to access a specific shard, so yes this shouldn't be private